addons/web/static/src/views/relational_model.js

```diff
@@ -2440,27 +2442,30 @@ export class DynamicGroupList extends DynamicList {
 
     /**
      * @param {string} groupName
+     * @param {boolean} [isFolded]
+     * @param {{ [fieldName: string]: any }} [additionalData]
      * @returns {Promise<Group>}
      */
-    async _createGroup(groupName) {
-        const [id, displayName] = await this.model.orm.call(
-            this.groupByField.relation,
-            "name_create",
-            [groupName],
-            { context: this.context }
-        );
+    async _createGroup(groupName, isFolded = false, additionalData = {}) {
+        const data = { ...additionalData, name: groupName };
+        if (isFolded) {
```

Comment:
@brboi  i added this check as i got an error when adding a model that don't have fold field like milestone

Reply:
Oh you're right here, nice catch.
But the condition should instead be:
```js
if(this.foldField) {
    data[this.foldField] = isFolded;
}
```

Reply:
I'll discuss the changes in web today with my team and will keep you in touch ;-)

Reply:
okay thank you for your time